FEM/invariants.h

```diff
@@ -0,0 +1,62 @@
+#ifndef INVARIANTS_H
+#define INVARIANTS_H
+
+#include <cfloat>
+#include "Eigen/Dense"
+// FEM-Makros
+#include "makros.h"
+#include "tools.h"
+
+namespace SolidMath
+{
+typedef Eigen::Matrix<double, 6, 1> KVec;
+
+Eigen::Matrix<double, 6, 6> Initialise_ident();
```

Comment:
Similar to KVec (line 12), you can also create an alias  for ``Eigen::Matrix<double, 6, 6>``.